app/controllers/RootController.scala

```diff
@@ -22,9 +22,9 @@ import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 
 import javax.inject.Inject
 
-class StartController @Inject()(mcc: MessagesControllerComponents) extends FrontendController(mcc) with I18nSupport {
+class RootController @Inject()(mcc: MessagesControllerComponents) extends FrontendController(mcc) with I18nSupport {
 
-  def displayStartPage(): Action[AnyContent] = Action {
+  def displayPage(): Action[AnyContent] = Action {
```

Comment:
NIT: doesn't actually display a page, as you're renaming it you could of picked something like 'redirectTo' :slightly_smiling_face: